src/js/content_scripts/input-mapper.js

```diff
@@ -124,9 +124,9 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                         invert: false
                     },
                     "3": {
-                        defaultAction: null,
+                        defaultAction: "analogTabbing",
```

Comment:
The analog tabbing should probably be bound to buttons 6 and 7, which are the only analog controls on PS4 dualshock and Xbox One X controllers.

Reply:
I'd also ask that you put the binding definitions in numeric order, it makes reviewing the list easier.

Reply:
`analogTabbing` was meant for joysticks, as when the user wants to tab using joystick. For the buttons, I've added `forwardTabbing` and `reverseTabbing`. I think I made a mistake with the function naming. I'll change it then.

Reply:
You're going to have to thing about the problem in general of making actions for:

1. Digital button.
2. Analogue buttons such as a triggers.
3. Axes.

The only difference between analogue buttons and axes is that axes have their own direction.  I think you can (and should) make it possible for analogue buttons to be used for anything that axes can be used for.  After all, if you allow people to remap controls but they can only map certain actions to the axes, that's more limiting than allowing them to assign some actions to analogue buttons.  We can leave a TODO for now, but this will really be important once you have a configuration interface.

Reply:
Actually I've been writing the functions for buttons in such a way that we can use it for triggers as well. For example, the `cutoffValue` that I set up at certain places was put in regard to these triggers.

Reply:
Moreover, in another light, you can also use the functions written for axes in triggers. If we take this approach, we just have to put an additional `invert` option to their configuration (as we did in axes) so that the user can choose to have the inverted action.